packages/server/controllers/events.js

```diff
@@ -47,6 +47,52 @@ async function getEvents(limit, action, eventId) {
   return events;
 }
 
+async function getPublicUpcomingEvents(limit, action, eventId) {
+  let events;
```

Comment:
Like my earlier comment about `action` being unnecessary because we're only going forward, the parameters can be reduced to just
```js
getPublicUpcomingEvents(limit, eventId)
```